OpenRA.Mods.Common/UpdateRules/Rules/20200503/RemoveBuildingInfluence.cs

```diff
@@ -0,0 +1,34 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2020 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.Collections.Generic;
+
+namespace OpenRA.Mods.Common.UpdateRules.Rules
+{
+	public class RemoveBuildingInfluence : UpdateRule
+	{
+		public override string Name { get { return "BuildingInfluence trait has been removed."; } }
+
+		public override string Description
+		{
+			get
+			{
+				return "BuildingInfluence trait has been removed. Its functionality has been integrated into ActorMap.";
```

Comment:
Since, AFAIK, `ActorMap` is not something modders are (or need to be) familiar with, would it be acceptable to tack something like "no action required on your part" onto the end of this?

Or "the engine now handles this internally," something like that.
I suspect this may cause confusion as-is.  Feel free to ignore.

Reply:
ActorMap actually is a trait nowadays. These descriptions are general info that is only displayed if the person running the utility specifically ask for it (using the `--detailed` arg) - not requiring explicit action is covered by the rule not yielding any specific manual messages.